docs/source/app_developers_guide/environment_setup.rst

```diff
@@ -0,0 +1,179 @@
+
+.. _environment_setup:
+
+*****************
+Environment Setup
+*****************
+
+Overview
+========
+
+This tutorial walks through the process of setting up Hyperledger Sawtooth for
+the purposes of app development. If you want to set up your environment for
+core development, please continue with the :doc:`/core_developers_guide`.
+
+At the end, you will be ready to perform app development tasks, such as
+creating new transaction families or handlers as described in the
+:ref:`transaction-family-tutorial`.
+
+Commands in this tutorial can be run via Terminal.app on MacOS, Git Bash on
+Windows, etc.
+
+
+Installing Sawtooth on Ubuntu Linux
+===================================
+
+You can install Sawtooth directly on your Ubuntu Linux machine, using the following steps:
+
+Prerequisites
+-------------
+
+Ubuntu 16.04 or later
+
+
+Installation
+------------
+
+Run the following commands from a terminal window, as root or with `sudo`:
+
+.. code-block:: console
+
+  $ echo "deb http://repo.sawtooth.me/ubuntu/0.8/stable xenial universe" >> /etc/apt/sources.list
+  $ apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 6B58B1AC10FB5F63
+  $ apt-get update && apt-get install -y sawtooth
+
+
+Next Steps
+----------
+
+Continue on to :doc:`intro_to_sawtooth`
+
+
+Installing Sawtooth Using Docker Compose 
+========================================
+
+
+Docker Instructions
+===================
+
+Prerequisites
+-------------
+
+The following tools are required:
+
+* `Docker Engine <https://docs.docker.com/engine/installation/>`_ (17.03.0-ce
+  or later)
+* `Docker Compose <https://docs.docker.com/compose/install/>`_ (Linux only)
+
+
+Step One: Clone Repository
+--------------------------
```

Comment:
Yes, I will create a story to store duplicated content in a single location, and insert it inline.